packages/fx-core/templates/plugins/resource/identity/bicep/provision.template.bicep

```diff
@@ -11,4 +11,5 @@ output identityOutput object = {
   identityName: userAssignedIdentityProvision.outputs.identityName
   identityResourceId: userAssignedIdentityProvision.outputs.identityResourceId
   identityClientId: userAssignedIdentityProvision.outputs.identityClientId
+  identityPrincipalId: userAssignedIdentityProvision.outputs.identityPrincipalId
```

Comment:
do we need to persist it to state.dev.json? It seems just used inside ARM template (bicep).

Reply:
It is fixed.